docs/csharp/language-reference/keywords/integral-types-table.md

```diff
@@ -1,36 +1,40 @@
 ---
-title: "Integral Types Table (C# Reference)"
-ms.date: 07/20/2015
+title: "Integral types table (C# Reference)"
+description: "Overview of the built-in C# integral types"
+ms.date: 08/20/2018
 helpviewer_keywords: 
   - "integral types, C#"
   - "Visual C#, integral types"
   - "types [C#], integral types"
   - "ranges of integral types [C#]"
 ms.assetid: 62e86126-46ff-40b0-9028-e61d7558268c
 ---
-# Integral Types Table (C# Reference)
+# Integral types table (C# Reference)
+
 The following table shows the sizes and ranges of the integral types, which constitute a subset of simple types.  
   
 |Type|Range|Size|  
 |----------|-----------|----------|  
-|[sbyte](../../../csharp/language-reference/keywords/sbyte.md)|-128 to 127|Signed 8-bit integer|  
-|[byte](../../../csharp/language-reference/keywords/byte.md)|0 to 255|Unsigned 8-bit integer|  
-|[char](../../../csharp/language-reference/keywords/char.md)|U+0000 to U+ffff|Unicode 16-bit character|  
-|[short](../../../csharp/language-reference/keywords/short.md)|-32,768 to 32,767|Signed 16-bit integer|  
-|[ushort](../../../csharp/language-reference/keywords/ushort.md)|0 to 65,535|Unsigned 16-bit integer|  
-|[int](../../../csharp/language-reference/keywords/int.md)|-2,147,483,648 to 2,147,483,647|Signed 32-bit integer|  
-|[uint](../../../csharp/language-reference/keywords/uint.md)|0 to 4,294,967,295|Unsigned 32-bit integer|  
-|[long](../../../csharp/language-reference/keywords/long.md)|-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807|Signed 64-bit integer|  
-|[ulong](../../../csharp/language-reference/keywords/ulong.md)|0 to 18,446,744,073,709,551,615|Unsigned 64-bit integer|  
+|[sbyte](sbyte.md)|-128 to 127|Signed 8-bit integer|  
+|[byte](byte.md)|0 to 255|Unsigned 8-bit integer|  
+|[char](char.md)|U+0000 to U+ffff|Unicode 16-bit character|  
+|[short](short.md)|-32,768 to 32,767|Signed 16-bit integer|  
+|[ushort](ushort.md)|0 to 65,535|Unsigned 16-bit integer|  
+|[int](int.md)|-2,147,483,648 to 2,147,483,647|Signed 32-bit integer|  
+|[uint](uint.md)|0 to 4,294,967,295|Unsigned 32-bit integer|  
+|[long](long.md)|-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807|Signed 64-bit integer|  
+|[ulong](ulong.md)|0 to 18,446,744,073,709,551,615|Unsigned 64-bit integer|  
   
- If the value represented by an integer literal exceeds the range of `ulong`, a compilation error will occur.  
+ If the value represented by an integer literal exceeds <xref:System.UInt64.MaxValue?displayProperty=nameWithType>, a compile error [CS1021](../../misc/cs1021.md) occurs.
```

Comment:
It would be useful here to note that BigInteger has no practical upper limit -- something like, "To work around this issue, you can use the \<System.Numerics.BigInteger> type, which has no upper bound."

Reply:
I've moved the `BigInteger` mention from the **See also** section to a newly created **Remarks** section as a separate paragraph.